Update key docs page to cover all protocols supported by Teleport (#10112)
gravitational/teleport: docs/pages/installation.mdx

---
title: Installing Teleport

> description: The guide for installing Teleport providing secure access to your servers, apps, dbs and desktops.

Reviewer: ```suggestion
description: How to install Teleport to get secure access to your servers, apps, 
 databases, and desktops.
```
